draft-ietf-modpod-group-processes.md

```diff
@@ -271,8 +274,12 @@ forum **and** the moderators.
 - **Administrators** are primarily responsible for managing their fora in
 accordance with guidance developed by the moderators and approved by
 the IESG. As such, they shall address reports of disruptive behavior
-in a timely fashion, apprising moderators of their disposition. For a
-Working Group, the chairs should perform moderation in a way that
+in a timely fashion, apprising moderators of reports or actions taken.
+
+  For a Working Group, the chairs are assumed to be adminstrators, though
+others might be assigned the role in addition to or instead of chairs
+with the assent of the responsible area director.
+Working Group fora administrators should perform moderation in a way that
```

Comment:
```suggestion
For a working group, chairs are by default the administrators.  They may
delegate this responsibility, but it must always transparently communicated
who holds this role and is hence taking moderation actions on behalf of the chairs.
Forum administrators should perform moderation in a way that
```

Reply:
That's not sufficient.  Chairs must take responsibility for receiving complaints.  That cannot be delegated.  How they are processed can be.

Reply:
Does what I wrote imply that is not so? (I don't see it.)

Reply:
If it is implied it is oblique at best.  Let's be unambiguous.  How about:
```suggestion
For a working group, chairs are by default the administrators.  They may
delegate this responsibility, but they must always accept, acknowledge,
and keep track of resolution of complaints of disruptive behavior.
Forum administrators should perform moderation in a way that
```